app/views/hosts/_compute_detail.html.erb

```diff
@@ -4,15 +4,5 @@
 <%= render :partial => provider_partial(compute_resource, 'base'),
             :locals => { :f => f, :compute_resource => compute_resource, :new_host => new_host }.merge(args_for_compute_resource_partial(@host)) %>
 
-<!--NICS-->
```

Comment:
The VMWare UI looks weird if this is still in. And the form fields have been moved a long time ago to the interfaces tab. So I think, the note is not needed anymore.